editor/project_manager.cpp

```diff
@@ -53,6 +53,8 @@
 #include "scene/main/window.h"
 #include "servers/display_server.h"
 
+static const String META_DRIVER_NAME = "driver_name";
```

Comment:
Meta keys are actually `StringName`, so if we're caching it, it would be good to avoid conversions :)

Otherwise you can just use `SNAME("driver_name")` in the three call places, I'm not sure the constant is really needed.

Reply:
I added the constant to avoid using string-typing (which `SNAME()` doesn't avoid). We use this in the editor about dialog too: https://github.com/godotengine/godot/blob/6d5c1086c3cc5442b4b8f7c752c941a0ce3150c5/editor/editor_about.cpp#L40-L41

I'll see if I can change it to StringName, but I'm not sure if it works in a `static const` context (I remember that not working in header files at least).